Sources/ITwinMobile/ITMApplication.swift

```diff
@@ -99,6 +108,17 @@ open class ITMApplication: NSObject, WKUIDelegate, WKNavigationDelegate {
             }
             return Promise.value(())
         }
+        configData = loadITMAppConfig()
+        if let configData = configData {
+            extractConfigDataToEnv(configData: configData)
+            if configData.isYes("ITMAPPLICATION_MESSAGE_LOGGING") {
```

Comment:
ITMAPPLICATION_MESSAGE_LOGGING and ITMAPPLICATION_FULL_MESSAGE_LOGGING should be documented in IOSSamples.xcconfig.

Reply:
Yes, that file is in a different repo.

Reply:
Oops.